app/routes/_dashboard_.tags.$tag.tsx

```diff
@@ -0,0 +1,53 @@
+import { useLoaderData, useRouteError } from "@remix-run/react";
+import invariant from "tiny-invariant";
+import type { LoaderArgs } from "@remix-run/node";
+import { json } from "@remix-run/node";
+import { searchContent } from "~/models/content.server";
+import CardWithContent from "~/components/card/CardWithContent";
+import TagButton from "~/components/buttons/TagButton";
+import ErrorComponent from "~/components/Error";
+
+export const loader = async ({ params }: LoaderArgs) => {
+  invariant(params.tag, "Tag not found");
+
+  const tag = params.tag;
+  const contentByTag = await searchContent(tag);
+
+  if (!contentByTag) {
+    throw new Response("Content by tag not found", { status: 404 });
+  }
+
+  return json({ contentByTag, tag });
+};
+
+const Tag = () => {
+  const { contentByTag, tag } = useLoaderData<typeof loader>();
+
+  return (
+    <div>
+      <section className="flex text-white">
+        <p className="inline-flex font-serif text-xl">
+          Viser alt innhold relatert til temaet
+        </p>
+        <TagButton className="ml-4" text={tag} notClickable />
+      </section>
+      <section className="mt-8 grid grid-cols-1 gap-4 sm:grid-cols-2 md:grid-cols-3">
+        {contentByTag.map((content) => (
```

Comment:
Hva vil skje om `contentByTag` ikke inneholder noe? Husker ikke om den returnerer `undefined` eller en tom liste.

Reply:
Godt spørsmål! Måtte sjekke 😁 
<img width="1225" alt="Skjermbilde 2023-05-04 kl  10 20 19" src="https://user-images.githubusercontent.com/6595982/236149099-a891d38d-1741-450d-a53e-a670072d9000.png">

Reply:
Burde kanskje heller rendre `Fant ikke noe innhold for {tag}`?

Reply:
Se der ja! Ja, tror det kan være lurt ja, så ikke brukeren "venter" på et resultat 😄

Reply:
En annen sjekk også om du gidder! Vet ikke om url er case sensitive; `tags/REACt`

Reply:
<img width="387" alt="Skjermbilde 2023-05-04 kl  10 36 14" src="https://user-images.githubusercontent.com/6595982/236153092-8ac4a6db-7989-44c4-b0af-9cf06ecb0d55.png">

Reply:
Gråtefjes 😢 Den synes jeg var fin!

Reply:
Går fint å skrive både css og CSS - sjekken skjer på backend, så det er null stress ☺️

Reply:
Perfetto!